include/babeltrace/debuginfo.h

```diff
@@ -0,0 +1,29 @@
+#ifndef _BABELTRACE_DEBUGINFO_H
+#define _BABELTRACE_DEBUGINFO_H
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <stddef.h>
+
+struct debug_info_source {
+	/* Strings are owned by this */
+	char *func;
+	long unsigned int line_no;
```

Comment:
Prefer using fixed-size types (e.g. uint64_t) when overflow may occur on word-size types.
However, a 32-bit unsigned integer should be enough for files shorter than 4,294,967,295 lines. :-)